src/coord/src/coord.rs

```diff
@@ -2021,6 +2025,18 @@ impl Coordinator {
         }
     }
 
+    async fn sequence_alter_compute_instance(
+        &mut self,
+        plan: AlterComputeInstancePlan,
+    ) -> Result<ExecuteResponse, CoordError> {
+        // TODO(benesch): update catalog with new configuration.
+
+        // TODO(mcsherry?): frob the controller with the diff of the compute
+        // instance configuration (`plan.config`).
```

Comment:
Do we need to do anything here? We could just ship the new instance plan at the controller and have it sort things out (i.e. do the diff there, as it is the source of truth of what is installed).